pkg/client/vcwallet/profile.go

```diff
@@ -25,6 +27,9 @@ var ErrProfileNotFound = errors.New("profile does not exist")
 
 // profile of VC wallet contains wallet specific settings of wallet user to be remembered.
 type profile struct {
+	// ID unique identifier assigned to this wallet profile.
+	ID string
+
 	// User ID of the wallet profile user.
 	User string
```

Comment:
I'm confused about the difference between `ID` and `User`. Based on the comments they sort of sound like the same thing? Maybe you can clarify the comments

Reply:
assigning UUID for each user profile which will be used for db namespacing, based on comments from previous PR

Reply:
You should add it to the comment so future readers know - if I was looking at that code I wouldn't have any idea what the difference is

Reply:
Or use different names or something

Reply:
Talked to @sudeshrshetty about it - I misread the code - please disregard my original concern!